### DISCUSSION TITLE
怎么使用injector.dll来注入

### DISCUSSION THREAD

**Discussion by chengxiangtong** [ttttupup/wxhelper#403] [Q&A]
看了相关的资料，注入工具有三个，1、命令行的方式，使用ConsoleApplication.exe；2、使用MFCApplication.exe视窗界面的方式；3、使用injector.dll来注入，很个版本都带有这个dll，但是没有看到过怎么使用这个dll，有知道的老铁吗？

**Comment by 234685117** [comment]
无论命令行的方式ConsoleApplication.exe还是MFCApplication.exe视窗界面的方式都是调用injector.dll的两个函数InjectDll和UnInjectDll。
